### ISSUE TITLE
Tag Background Color

### ISSUE THREAD

**Issue by prionkor** [toystars/react-native-multiple-select#100]
## Issue summary
There is no option for tag background color right now. Please introduce this option

**Comment by satyabrata45** [comment]
yes Please introduce an option for tag background color and tag position
